src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -3901,6 +3901,13 @@ void AuraEffect::HandleModCastingSpeed(AuraApplication const* aurApp, uint8 mode
         return;
     }
 
+    int32 spellGroupVal = target->GetHighestExclusiveSameEffectSpellGroupValue(this, GetAuraType());
+    if (abs(spellGroupVal) >= abs(GetAmount()))
```

Comment:
I would prefer to use std::abs() rather than abs()

Reply:
all other handlers is using abs()
Need geral change then 💃